unitTests/ui/FieldTableTest.ts

```diff
@@ -42,7 +55,7 @@ export function FieldTableTest() {
         });
 
         it('should not wrap table in a toggle container', function() {
-          expect($$(test.env.element).find('.coveo-field-table-toggle-container')).toBeNull();
+          expect(findToggleContainer()).toBeNull();
```

Comment:
I added a function to retrieve the toggle container in my PR 😄 I would appreciate if you could merge master and then remove my function and update references to use yours please. Thanks!

Reply:
Sweet! I'll use your function instead.